src/ImageCropBundle/Bridge/Chameleon/Field/TCMSFieldMediaWithImageCrop.php

```diff
@@ -38,6 +39,7 @@ class TCMSFieldMediaWithImageCrop extends TCMSFieldExtendedLookupMedia
      */
     public function ChangeFieldDefinition($sOldName, $sNewName, &$postData = null)
     {
+        $originalDefinitionData = $this->oDefinition->sqlData;
         parent::ChangeFieldDefinition($sOldName, $sNewName, $postData);
 
         $additionalFieldNameOldFieldName = $this->getFieldNameOfAdditionalField($sOldName);
```

Comment:
When trying this (with current unfixed code) I also noticed that I cannot even create a field of that type newly.
There will be a db error:

'ALTER TABLE `shop_payment_handler_parameter` CHANGE `new_field_image_crop_id` `foo_cms_media_id_image_crop_id` CHAR(36) CHARACTER SET latin1 COLLATE latin1_general_ci NOT NULL':  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'new_field_image_crop_id'

Reply:
Fixed the problem.